Automattic/wp-calypso
# Combine WordAds Earnings + Settings into a single page

#### Changes proposed in this Pull Request

* Combine Ads Earnings & Settings UI into a single display.
* Put ads settings into 2 collapsible cards:
  1. WordAds Account (including Paypal email)
  2. Placement Settings (including whether ads are turned on or off)
* Sites will an active account will have the account card collapsed by default, and the placements card expanded by default. The earnings tables will always display below these.

I left the scaffolding in place for tabs in case one of the other sections will use that in the future.

#### Testing instructions

Lets start with testing around the UI, make sure everything makes sense and we're happy with the overall direction. Then we can dive into the technicals and likely add some more cleanup.

Here's what you should see while testing:

##### Application view
<img width="797" alt="Screen Shot 2019-10-22 at 4 08 48 PM" src="https://user-images.githubusercontent.com/2522431/67328018-d5e94200-f4e6-11e9-9bb9-b4af6bbb3a66.png">

##### WordPress.com Site view
<img width="795" alt="Screen Shot 2019-10-22 at 4 08 02 PM" src="https://user-images.githubusercontent.com/2522431/67328008-d1bd2480-f4e6-11e9-988d-8d7c47e5b4e2.png">

##### Jetpack site view
<img width="797" alt="Screen Shot 2019-10-22 at 4 08 30 PM" src="https://user-images.githubusercontent.com/2522431/67328011-d386e800-f4e6-11e9-9711-7491f3e79722.png">
